lib/simp/rake/pupmod/helpers.rb

```diff
@@ -352,7 +353,7 @@ def custom_fixtures_hook(opts = {
           # determine mission-impacting files that have changed
           files_changed = `git diff tags/#{last_tag} --name-only`.strip.split("\n")
           files_changed.delete_if do |file|
-            file[0] ==  '.' or file =~ /^Gemfile/ or file == 'Rakefile' or file =~/^spec\// or file =~/^doc/
+            file[0] ==  '.' or file =~ /^Gemfile/ or file == 'Rakefile' or file =~/^spec\// or file =~/^doc/ or file =~/^rakelib/
```

Comment:
Is there a reason we are not checking for exact file name in all cases? It is doing so for 'Rakefile', but all others are open to possible false-positive matches with the regex due to no 'end of line' '$'.

(same comment for the regex checks in the other files)